CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+2.0.0 (unreleased)
+------------------
+
+* **2016-12-03**: [BC BREAK] Remove sonata admin related classes and configuration
```

Comment:
indention of the leading `*` is off.

and i'd prefer to say "Moved sonata admin ... to the CmfSonataAdminIntegrationBundle"